mail_optional_follower_notification/models/mail_thread.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _notify_compute_recipients(self, message, msg_vals):
+        """ Compute recipients to notify based on subtype and followers. This
+        method returns data structured as expected for ``_notify_recipients``. """
+        recipient_data = super()._notify_compute_recipients(message, msg_vals)
+        if "notify_followers" in self.env.context and not self.env.context.get(
```

Comment:
why not just `if self.env.context.get("notify_followers")`?

Reply:
@simahawk otherwise you change the default behaviour if the key is not explicitly provided into the context.
We could write `if self.env.get("notify_followers", True)`  but the actual syntax is also a way to say: "If 'notify_followers' is into the context I've to take car of this, otherwise I pass my way'

Reply:
if the key is None, very likely is not there. Not a blocker of course.